shared/src/commonMain/kotlin/org/hyperskill/app/step_completion/presentation/StepCompletionReducer.kt

```diff
@@ -59,8 +60,9 @@ class StepCompletionReducer(private val stepRoute: StepRoute) : StateReducer<Sta
             is Message.CheckTopicCompletionStatus.Completed ->
                 state.copy(
                     continueButtonAction = ContinueButtonAction.NavigateToHomeScreen,
-                    isPracticingLoading = false
-                ) to setOf(Action.ViewAction.ShowTopicCompletedModal(message.modalText))
+                    isPracticingLoading = false,
+                    nextStepRoute = message.nextStepId?.let { StepRoute.Learn(it) }
```

Comment:
Is it correct to set it as `nextStepRoute` -> `StepRoute.Learn`? Or we should use `StepCompletionReducer. stepRoute` but with a different step id?

Reply:
yes, it is correct, we show next topic after completion only in `Learn` step route